docs/data/material/getting-started/templates/templates.md

```diff
@@ -5,7 +5,7 @@ title: 9+ Free React Templates
 
 # React Templates
 
-<p class="description">A selection of free react templates to help you get started building your app. The collection contains react dashboard, react admin, and more.</p>
+<p class="description">A selection of free Material UI templates to help you get started building your app. The collection contains react dashboard, react admin, and more.</p>
 
 <!-- #default-branch-switch -->
 
```

Comment:
Adding an intro statement here to reiterate the page description and lead into the body text.

```suggestion

Our curated collection of Material UI templates includes dashboards, sign-in and sign-up pages, blog, checkout flow, and more.

```